### PR TITLE
[MAINT/DOC] Update information for nilearn organizational docs

### LINKED ISSUE #3329: Update organizational docs
I listed some supporting documentation that seems to need some updating. Let me know your feedback and feel free to add anything I may have missed.

### Affected pages

- Our [license](https://github.com/nilearn/nilearn/blob/main/LICENSE) (includes old sphinx and jquery licenses)
- Update core dev info and funding in https://nilearn.github.io/dev/authors.html (also see #3270)
  - Remove triage section? (if so, see also [here](http://nilearn.github.io/dev/development.html#join-the-triage-team))
- ~~Update 'Development version' installation instructions in [introduction](https://nilearn.github.io/dev/introduction.html#installing-nilearn)~~
- Add Code of Conduct

### PR DESCRIPTION
**Pull request by ymzayek**
Closes #3329 and #3270.
Addresses #3264 

Changes proposed in this pull request:

- Update `LICENSE`
- Update team and funding in `AUTHORS.rst`
- Remove triage team
- Add reference to NIPY Code of Conduct

### CODE REVIEW

**Review by bthirion — APPROVED** [review]
Thx !

File: AUTHORS.rst [hunk 1]
```diff
@@ -23,31 +23,25 @@ Core developers
 
 The nilearn core developers are:
 
-* `Alexandre Gramfort`_
 * `Alexis Thual`_
 * `Bertrand Thirion`_
-* `Binh Nguyen`_
 * `Elizabeth DuPre`_
-* `Gael Varoquaux`_
```
Comment (jeromedockes) [inline AUTHORS.rst:31]: 😢

**Review by jeromedockes — APPROVED** [review]

File: AUTHORS.rst (hunk 1 above)
Comment (emdupre) [reply AUTHORS.rst:31]: End of an era !

**Comment by ymzayek** [comment]
I guess we can leave adding the CZI info for later when the new developer is hired. So this should be good to merge

**Comment by ymzayek** [comment]
Merging then

**Pull request merged**